frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.ts

```diff
@@ -119,7 +143,11 @@ export class ExperimentListComponent implements OnInit, OnDestroy, AfterViewInit
   applyFilter(filterValue: string) {
     this.filterExperimentPredicate(this.selectedExperimentFilterOption);
     if (filterValue !== undefined) {
```

Comment:
same as other comment, filterValue needs to be a string, not just 'not undefined', a null value can slip in here, for instance.

Reply:
yeah this was a very old code which used undefined. Nice catch, will update. :)